layers/meta-tegrademo/recipes-bsp/u-boot/u-boot-tegra/0014-common-bootm.c-Increase-CONFIG_SYS_BOOTM_LEN-to-64MB.patch

```diff
@@ -0,0 +1,27 @@
+From bdedff1c9c382222d813c8b5bf923d4e641cb392 Mon Sep 17 00:00:00 2001
+From: Ilies CHERGUI <ilies.chergui@gmail.com>
+Date: Tue, 15 Dec 2020 14:31:11 +0000
+Subject: [PATCH 1/2] common/bootm.c: Increase CONFIG_SYS_BOOTM_LEN to 64MB.
+
+Signed-off-by: Ilies CHERGUI <ilies.chergui@gmail.com>
+---
+ common/bootm.c | 3 +--
+ 1 file changed, 1 insertion(+), 2 deletions(-)
+
+diff --git a/common/bootm.c b/common/bootm.c
+index db4362a643..1e2d0799b9 100644
+--- a/common/bootm.c
++++ b/common/bootm.c
+@@ -28,8 +28,7 @@
+ #include <image.h>
+ 
+ #ifndef CONFIG_SYS_BOOTM_LEN
+-/* use 8MByte as default max gunzip size */
+-#define CONFIG_SYS_BOOTM_LEN	0x800000
++#define CONFIG_SYS_BOOTM_LEN (64 << 20) /* 64MB */
```

Comment:
This definition should probably go in a target-specific header in `include/configs`, rather than in the U-Boot common code.  That seems to be the pattern used for other targets in the U-Boot code.

Reply:
I will try it and let you know

Reply:
The change as you suggest is Perfect. 
Its make sense to update the target-specific header.